_posts/2016-07-05-Open-Sidewalks-reflection.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+author: Open Sidewalks Team
+date:   2016-07-05
+Learning How to Design as Allies Not Community Members.
```

Comment:
For future reference, this should be: 

    title: Learning How to Design as Allies Not Community Members

(note the `title`). I'll fix this with a quick commit on my side.